docs/source/api/v2/ping.rst

```diff
@@ -0,0 +1,62 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-ping:
+
+********
+``ping``
+********
+Checks whether Traffic Ops is online.
+
+``GET``
+=======
+:Auth. Required: No
+:Response Type:  Object
```

Comment:
Response type is actually "<code>\`\`undefined\`\`</code>" since there's no `response` key.

Reply:
Response type changed in 8fe4ab7d06.